Mixmatch/streetview_dataset/parse_TFRecord.ipynb

```diff
@@ -0,0 +1,221 @@
+    "            elif confidence.size==0 or (confidence.size>0 and max(confidence)<0.1):\n",
+    "                label = 0\n",
+    "            else:\n",
+    "                continue\n",
+    "\n",
+    "            img_path = os.path.join(folder_path, img_name)\n",
+    "            try:\n",
+    "                img = Image.open(img_path, \"r\")\n",
+    "\n",
+    "                # Exclude all non RGB images\n",
+    "                if len(img.getbands()) != 3:\n",
+    "                    continue\n",
+    "\n",
+    "                if label:\n",
+    "                    img = img.crop(bbox)\n",
+    "                img = img.resize((64, 64))\n",
+    "\n",
+    "                example = img2example(img, label)\n",
+    "                writer.write(example.SerializeToString())\n",
+    "            except Exception as e:\n",
+    "                print(e)\n",
+    "                print(img_path + \" is not valid\")\n",
+    "\n",
+    "def write_TFRecord_TC11(folder_path, writer):\n",
+    "    for img_name in os.listdir(folder_path):\n",
+    "        label = 1\n",
+    "        img_path = os.path.join(folder_path, img_name)\n",
+    "        \n",
+    "        try:\n",
+    "            img = Image.open(img_path, \"r\")\n",
+    "\n",
+    "            # Exclude all non RGB images\n",
+    "            if len(img.getbands()) != 3:\n",
+    "                continue\n",
+    "            \n",
+    "            img = img.resize((64, 64))\n",
+    "\n",
+    "            example = img2example(img, label)\n",
+    "            writer.write(example.SerializeToString())\n",
+    "        except Exception as e:\n",
+    "            print(e)\n",
+    "            print(img_path + \" is not valid\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# input tfrecord file path\n",
+    "input_record_path = './streetlearn-detections/'\n",
+    "UCF_img_dir = './UCF_Streetview_Dataset/raw/'\n",
+    "TC11_img_dir = './TC11/svt1'\n",
+    "\n",
+    "# name format of the output tfrecord files\n",
+    "output_record_path = \"../ML_DATA/\"\n",
+    "train_record_fileName = \"streetview-train.tfrecord\"\n",
+    "test_record_fileName = \"streetview-test.tfrecord\"\n",
+    "\n",
+    "# tfrecord file writer\n",
+    "train_writer = tf.io.TFRecordWriter(output_record_path + train_record_fileName)\n",
+    "test_writer = tf.io.TFRecordWriter(output_record_path + test_record_fileName)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write train and test tfrecord for TC11 dataset\n",
+    "train_image_path = os.path.join(TC11_img_dir, 'train')\n",
+    "test_image_path = os.path.join(TC11_img_dir, 'test')\n",
+    "\n",
+    "write_TFRecord_TC11(train_image_path, train_writer)\n",
+    "write_TFRecord_TC11(test_image_path, test_writer)"
```

Comment:
I am confused with this part. Do you write TC11 dataset to both train set and test set?

Reply:
This PR was created last week, at that time TC11 is written into both train and test set for this version 1 dataset.
Do you want me to update to a newer version? Or keep this version for this PR and the updated other versions of the dataset will be created in future PRs.

For me, I think it's better to keep this PR as it is right now, cause future versions of the dataset don't only change this part of the code. But it's up to you for the final decision.

Reply:
That makes sense to me. It's fine to keep this version in this PR.